### PR TITLE
Bug 1584119 - Add link to code coverage frontend in 'Other Tools' panel

### PR DESCRIPTION
**Pull request by marco-c**

### CODE REVIEW

File: tools/src/bin/output-file.rs [hunk 1]
```diff
@@ -266,6 +266,14 @@ fn main() {
                 accel_key: None,
             });
         }
+        if let Some(ref ccov_root) = tree_config.paths.ccov_root {
+            tools_items.push(PanelItem {
+                title: "Code Coverage".to_owned(),
+                link: format!("{}#revision=latest&path={}", ccov_root, path),
```
Comment (emilio) [inline tools/src/bin/output-file.rs]: Should this be revision-aware rather than assuming `latest`?

Comment (marco-c) [reply tools/src/bin/output-file.rs]: We could do that, but right now the Mercurial revision is not available here (we only have the git one).
We could use the mapper, but I think we can do that in a follow-up (adding support for HG Web and DXR too, which are both pointing to "tip").

Comment (emilio) [reply tools/src/bin/output-file.rs]: Sure, sounds good.

Comment (marco-c) [reply tools/src/bin/output-file.rs]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1585266.

**Pull request merged**